lib/plugins/babelRuntimePathResolver.js

```diff
@@ -2,11 +2,12 @@
 // in OBT's node_modules directory and not the module's
 'use strict';
 
-const path = require('path');
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-const rootDirectory = path.resolve(__dirname, '../../node_modules');
+
+// HACK: Can't require.resolve babel-runtime so use babel-core instead
+const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
```

Comment:
`require.resolve` gives you the path to the main **file**, so in this something like this:-

`/path/to/my/project/node_modules/babel-core/main.js`

The replace will get rid of everything **after** the last `node_modules`

```js
'/path/to/node_modules/thing/main.js'.replace(/(.*node_modules).*/, '$1');
// => /path/to/thing/node_modules

'/path/to/node_modules/thing/node_modules/other-thing/main.js'.replace(/(.*node_modules).*/, '$1');
// => /path/to/node_modules/thing/node_modules

'/path/to/node_modules/thing/node_modules/other-thing/subdirectory/main-in-a-sub-dir.js'.replace(/(.*node_modules).*/, '$1');
// => /path/to/node_modules/thing/node_modules

```